bpf/lib/nodeport.h

```diff
@@ -753,7 +756,11 @@ static __always_inline int nodeport_lb6(struct __ctx_buff *ctx,
 
 	svc = lb6_lookup_service(&key, false);
 	if (svc) {
-		const bool skip_l3_xlate = DSR_ENCAP_MODE == DSR_ENCAP_IPIP;
+#if DSR_ENCAP_MODE == DSR_ENCAP_IPIP
+		const bool skip_l3_xlate = true;
+#else
+		const bool skip_l3_xlate = false;
+#endif
```

Comment:
```suggestion
		const bool skip_l3_xlate = DSR_ENCAP_MODE == DSR_ENCAP_IPIP;
```

Reply:
I first attempt is the same as you suggested. But I got following error and had a hard time to figure out why.

/work/go/src/github.com/cilium/cilium/bpf/lib/nodeport.h:766:48: error: use of undeclared identifier 'DSR_ENCAP_IPIP'
                const bool skip_l3_xlate = DSR_ENCAP_MODE == DSR_ENCAP_IPIP;

So I used current way to make the compiler happy.